css/global.css

```diff
@@ -10,4 +10,8 @@
 
 .c-message_kit__hidden_message_blur {
   filter: blur(0) !important;
+}
+
+.c-message_kit__hidden_message_blur * {
```

Comment:
I learned a bit about the asterisk in CSS while working on this PR. [This Stack Overflow post](https://stackoverflow.com/a/1204290) specifies exactly how the * functions when following a different CSS selector. It selects all HTML elements with a parent element with the class `c-message_kit__hidden_message_blur`.